common/client-libs/validator-client/src/nymd/fee_helpers.rs

```diff
@@ -0,0 +1,88 @@
+// Copyright 2021 - Nym Technologies SA <contact@nymtech.net>
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::nymd::GasPrice;
+use cosmos_sdk::tx::{Fee, Gas};
+use cosmos_sdk::Coin;
+use cosmwasm_std::Uint128;
+
+#[derive(Debug, Copy, Clone, Ord, PartialOrd, Eq, PartialEq, Hash)]
+pub enum Operation {
+    Upload,
+    Init,
+    Migrate,
+    ChangeAdmin,
+    Send,
+
+    BondMixnode,
+}
+
+pub(crate) fn calculate_fee(gas_price: &GasPrice, gas_limit: Gas) -> Coin {
+    let limit_uint128 = Uint128::from(gas_limit.value());
+    let amount = gas_price.amount * limit_uint128;
+    assert!(amount.u128() <= u64::MAX as u128);
+    Coin {
+        denom: gas_price.denom.clone(),
+        amount: (amount.u128() as u64).into(),
+    }
+}
+
+impl Operation {
+    pub(crate) fn default_gas_limit(&self) -> Gas {
+        match self {
+            Operation::Upload => 2_500_000u64.into(),
+            Operation::Init => 500_000u64.into(),
+            Operation::Migrate => 200_000u64.into(),
+            Operation::ChangeAdmin => 80_000u64.into(),
+            Operation::Send => 80_000u64.into(),
+
+            Operation::BondMixnode => 175_000u64.into(),
```

Comment:
Would it make sense to have those values as `const`s in one place, so we can modify them more easily?